.github/workflows/workflow.yml

```diff
@@ -127,3 +127,27 @@ jobs:
           SSH_CONFIG: ${{ secrets[env.SSH_CONFIG_SECRET_NAME] }}
           SSH_KEY: ${{ secrets[env.SSH_KEY_SECRET_NAME] }}
         run: ./.github/scripts/deploy.sh
+
+      - name: Login
+        if: env.TIER == 'PROD'
+        uses: atlassian/gajira-login@master
+        env:
+          JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
+          JIRA_USER_EMAIL: ${{ secrets.JIRA_USER_EMAIL }}
+          JIRA_API_TOKEN: ${{ secrets.JIRA_API_TOKEN }}
+
+      - name: Create Jira Transition List
+        if: env.TIER == 'PROD'
+        run: curl https://terraware.github.io/terraware-web/unreleased.log | grep -E 'SW-[0-9]+' -o | sort -u > ./docs/jiralist.txt
```

Comment:
We could - I *think* Jira would support lower case keys.  But I would also need to update the action then because I followed the default pattern here: https://confluence.atlassian.com/adminjiraserver/changing-the-project-key-format-938847081.html

If you think it's worth it lmk and I'll make the changes

Reply:
I was mostly thinking of cases where user typed 'Sw-1234' instead of 'SW-1234' in the git commit.
Completely missed the part where Jira needs to recognize Sw-1234 as well.
Def not worth looking into for now, my bad :)